version.txt

```diff
@@ -1,63 +1,63 @@
-v2.0.0
+v2.1.dev
 abalone_age.json,1.2.1
 amniote_life_hist.json,1.1.1
 antarctic_breed_bird.json,1.2.0
-aquatic_animal_excretion.py,1.0.0
-bioclim.py,1.1.0
-biomass_allometry_db.py,1.3.0
+aquatic_animal_excretion.py,1.1.0
+bioclim.py,1.2.0
+biomass_allometry_db.py,1.4.0
 bird_size.json,1.2.0
 breast_cancer_wi.json,1.1.1
-breed_bird_survey.py,1.3.1
-breed_bird_survey_50stop.py,1.3.1
+breed_bird_survey.py,1.4.0
+breed_bird_survey_50stop.py,1.4.0
 butterfly_population_network.json,1.2.0
 car_eval.json,1.1.1
 community_abundance_misc.json,1.0.1
 elton_traits.json,1.2.0
 fish_parasite_hosts.json,1.2.0
 forest_biomass_china.json,1.2.0
 forest_fires_portugal.json,1.1.3
-forest_inventory_analysis.py,1.3.1
+forest_inventory_analysis.py,1.4.0
 forest_plots_michigan.json,1.2.0
-forest_plots_wghats.py,1.2.0
+forest_plots_wghats.py,1.3.0
 fray_jorge_ecology.json,1.3.0
-gentry_forest_transects.py,1.3.1
+gentry_forest_transects.py,1.4.0
 home_ranges.json,1.1.1
-intertidal_abund_me.py,1.4.0
+intertidal_abund_me.py,1.5.0
 iris.json,1.2.0
-la_selva_trees.py,1.3.0
+la_selva_trees.py,1.4.0
 leaf_herbivory.json,1.2.0
 mammal_community_db.json,1.2.0
 mammal_diet.json,1.1.1
 mammal_life_hist.json,1.2.0
 mammal_masses.json,1.3.0
 mammal_metabolic_rate.json,1.2.0
-MammalSuperTree.py,1.1.0
+MammalSuperTree.py,1.2.0
 mapped_plant_quads_co.json,1.2.0
 mapped_plant_quads_id.json,1.2.0
 mapped_plant_quads_ks.json,1.2.0
 mapped_plant_quads_mt.json,1.2.0
 mt_st_helens_veg.json,1.2.0
 npn.py,2.0.0
```

Comment:
The best way to update version.txt file is clean up your repository. Run `git clean -fx` and then `retriever reset all`. This gives it a clean environment to run`python version.py`